### PR TITLE
Ad Layout & Style Sheet Property Selection

### PR DESCRIPTION
**Pull request by thomas-tosoni**
MEDIARITHMICS/core-platform-services/pull/116

### CODE REVIEW

File: app/src/core/adlayouts/ViewAllController.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ define(['./module', 'jquery'], function (module, $) {
 
   module.controller('core/adlayouts/ViewAllController', [
     '$scope', 'Restangular', 'core/common/auth/Session', '$location', '$state', '$stateParams', '$uibModal', '$log',
+<<<<<<< HEAD
```
Comment (mgrenonville) [inline app/src/core/adlayouts/ViewAllController.js]: Oups :)

Comment (thomas-tosoni) [inline app/src/core/adlayouts/ViewAllController.js]: Oups en effet :+1:

**Comment by mgrenonville** [comment]
#Create stylesheet plugin

curl -X POST "http://$MICS_SERVER/v1/plugins" -H 'Content-type: application/json' -H "Authorization: $MICS_AUTH"   --data '{"artifact_id":"style-sheet", "group_id":"com.mediarithmics.datafile.style_sheet","organisation_id":"1","plugin_type":"STYLE_SHEET", "version":"0"}'


 curl -X POST "http://$MICS_SERVER/v1/plugins/1013/versions" -H 'Content-type: application/json' -H "Authorization: $MICS_AUTH"  --data '{ "version_id":"1.0.0", "plugin_id":"XXX", "plugin_properties":[{ "origin":"PLUGIN", "value":{" value ": null }, "technical_name":"logo", "property_type" : "ASSET", "writable" :"true", "deletable" : "false" }]}' -v

**Comment by mgrenonville** [comment]
grunt error :  
app/src/core/common/properties/propertiesDirectives.js
  line 109  col 47  Did you mean to return a conditional instead of an assignment?

**Comment by thomas-tosoni** [comment]
JSHint fixed

**Pull request merged**